app/routes/index.js

```diff
@@ -28,9 +21,6 @@ export default Route.extend({
     },
 
     _emptyModels() {
-        return {
-            providers: [],
-            actionsList: [],
-        };
+        return [];
```

Comment:
There's no point to a method that just returns `[]`, please remove `_emptyModels` and use a literal `[]` instead.